cilium/cmd/monitor.go

```diff
@@ -235,35 +240,46 @@ func runMonitor() {
 	signal.Notify(signalChan, os.Interrupt)
 	go func() {
```

Comment:
Because of the jump you're doing on line 226, you're are also initializing this go routine